src/PartitionLoops.h

```diff
@@ -8,16 +8,17 @@
  */
 
 #include "Expr.h"
+#include "Scope.h"
 
 namespace Halide {
 namespace Internal {
 
 /** Return true if an expression uses a likely tag that isn't captured
  * by an enclosing Select, Min, or Max. */
-bool has_uncaptured_likely_tag(const Expr &e);
+bool has_uncaptured_likely_tag(const Expr &e, const Scope<> &scope);
```

Comment:
The comments need to be updated to say that the scope is for variables of lets which has likely tag in them.